docs/start/getting-started/fragments/next/api.md

```diff
@@ -0,0 +1,436 @@
+Now that you’ve created and configured a Next.js app and initialized a new Amplify project, you can add a feature. The first feature you will add is an API.
+
+The Amplify CLI supports creating and interacting with two types of API categories: REST and GraphQL.
+
+The API you will be creating in this step is a GraphQL API using AWS AppSync (a managed GraphQL service) and the database will be Amazon DynamoDB (a NoSQL database).
+
+## Create a GraphQL API and database
+
+Add a [GraphQL API](https://docs.aws.amazon.com/appsync/latest/devguide/designing-a-graphql-api.html) to your app and automatically provision a database by running the the following command from the root of your application directory:
+
+```bash
+amplify add api
+```
+
+Select the explicit values below to enable **IAM** (for public read access) and **Cognito User Pools** (for authenticated access).
+
+```console
+? Please select from one of the below mentioned services:
+# GraphQL
+? Provide API name:
+# nextamplified
+? Choose the default authorization type for the API:
+# API key
+? Enter a description for the API key:
+# 
+? After how many days from now the API key should expire (1-365):
+# 7
+? Do you want to configure advanced settings for the GraphQL API?
+#  Yes, I want to make some additional changes.
+? Configure additional auth types?
+# Yes
+? Choose the additional authorization types you want to configure for the API:
+# Amazon Cognito User Pool
+? Do you want to use the default authentication and security configuration?
+# Default configuration
+? How do you want users to be able to sign in?
+# Username
+? Do you want to configure advanced settings?
+# No, I am done.
+? Configure conflict detection?
+# No
+? Do you have an annotated GraphQL schema?
+# No
+? Do you want a guided schema creation?
```

Comment:
The CLI no longer asks this from what I can tell.

It jumps straight from "do you have an annotated GraphQL schema" to "choose a schema template" to "Do you want to edit the schema now?"

Reply:
Weird, I can re-capture this. I took this straight from my output.


Can you paste yours for comparison?

Reply:
Ah, my version is old!

Reply:
Updated via 59d87759af0005affc4ebb24a9b392d0cef3700d